code/modules/mob/mob.dm

```diff
@@ -446,9 +446,31 @@
 		// shift-click catcher may issue examinate() calls for out-of-sight turfs
 		return
 
-	if(is_blind())
-		to_chat(src, "<span class='warning'>Something is there but you can't see it!</span>")
-		return
+	if(is_blind()) //blind people see things differently (through touch)
+		//need to be next to something, awake, on help intent, and have an empty hand
+		if(!in_range(A, src) || incapacitated() || a_intent != INTENT_HELP || LAZYLEN(do_afters) >= get_num_arms() || get_active_held_item())
+			to_chat(src, "<span class='warning'>Something is there, but you can't see it!</span>")
```

Comment:
The messaging should probably be different based on *why* you can't examine it. As in, if I examine something with something in my hand, and it doesn't work, it's not obvious to me that I just need to switch to an empty one.